### ISSUE TITLE
no-lonely-if shouldn't complain when child if has else

### ISSUE THREAD

**Issue by geek** [eslint/eslint#1355]
```js
else {
        if () {
            ...
        }
        else {
            ...
        }
    }
```

Shouldn't cause an issue

**Comment by ilyavolodin** [comment]
Closing this, as it's been a while without follow-up. Feel free to reopen if it's still something you want.
